### ISSUE TITLE
feature request: toggle lights in configurator

### ISSUE THREAD

**Issue by claviger-pc** [jfedor2/hid-remapper#39]
Having the lights on is very useful for testing and pairing, but they are not necessary habitually. Would it be possible to toggle them off in the configurator (I am using a Seeed Xiao nrf). Perhaps there could simply be a no-light firmware variant.

Thanks for all of your work!

**Comment by fxliang** [comment]
is it about led indicator lights?I try it with ikbc c104, there are not any light actions, no caps lights, no numb locks lights.

**Comment by jfedor2** [comment]
No, this is about the lights on the boards running HID Remapper.

We do have an issue about getting CapsLock etc. to work. (#6)

**Comment by fxliang** [comment]
got it
